contrib/!lang/python/packages.el

```diff
@@ -162,16 +162,24 @@
 
       (defun python-setup-shell ()
         (if (executable-find "ipython")
-            (setq python-shell-interpreter "ipython"
-                  ;; python-shell-interpreter-args (if (system-is-mac)
-                  ;;                                   "--gui=osx --matplotlib=osx --colors=Linux"
-                  ;;                                 (if (system-is-linux)
-                  ;;                                     "--gui=wx --matplotlib=wx --colors=Linux"))
-                  python-shell-prompt-regexp "In \\[[0-9]+\\]: "
-                  python-shell-prompt-output-regexp "Out\\[[0-9]+\\]: "
-                  python-shell-completion-setup-code "from IPython.core.completerlib import module_completion"
-                  python-shell-completion-module-string-code "';'.join(module_completion('''%s'''))\n"
-                  python-shell-completion-string-code "';'.join(get_ipython().Completer.all_completions('''%s'''))\n")
+            (if (system-is-mswindows)
+                (setq python-shell-interpreter "python"
+                      python-shell-interpreter-args "-i c:\\python27\\scripts\\ipython-script.py"
```

Comment:
AFAIK any variable listed in the `:variables` keyword for a layer is available to use. See e.g. the `python-test-runner` variable https://github.com/syl20bnr/spacemacs/blob/develop/contrib/!lang/python/packages.el#L129

You should be able to introduce a variable and then just use it instead of the text. You also want to set a default like [this](https://github.com/syl20bnr/spacemacs/blob/develop/contrib/!lang/python/config.el#L22) in the config.el file.